packages/devtools_app/lib/src/treemap/treemap.dart

```diff
@@ -0,0 +1,459 @@
+import 'package:flutter/material.dart';
+
```

Comment:
instead of putting this inside its own treemap/ directory, let's put it inside src/charts/